sample/src/androidTest/java/com/airbnb/lottie/samples/FragmentVisibilityTests.kt

```diff
@@ -158,7 +158,7 @@ class FragmentVisibilityTests {
             }
         }
 
-        val scenario = launchFragmentInContainer<TestFragment>(themeResId = R.style.Theme_AppCompat)
+        val scenario = launchFragmentInContainer<TestFragment>(themeResId = AppcompatR.style.Theme_AppCompat)
```

Comment:
AGP starting from 8.0 has `android.nonTransitiveRClass` set to `true` by default https://developer.android.com/build/releases/gradle-plugin#default-changes